ambry-server/src/integration-test/java/com/github/ambry/server/CloudAndStoreReplicationTest.java

```diff
@@ -306,9 +306,8 @@ private void testGetOnServerNode(Map<BlobId, Integer> blobIdToSizeMap, DataNodeI
       assertEquals("Error in getting the recovered blobs", ServerErrorCode.No_Error,
           partitionResponseInfo.getErrorCode());
       //old value is 272. Adding 9 Bytes due to the two fields added 4 + 4 Blob Property BYTE, +1 for compression.
-      int putMessageSize = PutMessageFormatInputStream.useBlobFormatV3 ? 281 : 280;
       for (MessageInfo messageInfo : partitionResponseInfo.getMessageInfoList()) {
-        assertEquals(blobIdToSizeMap.get(messageInfo.getStoreKey()) + putMessageSize, messageInfo.getSize());
+        assertEquals(blobIdToSizeMap.get(messageInfo.getStoreKey()) + 281, messageInfo.getSize());
```

Comment:
can u add a comment why 281 ?

Reply:
The explanation is in the comment: old value is 272. Adding 9 Bytes due to the two fields added 4 + 4 Blob Property BYTE, +1 for compression.